### PR TITLE
fix: set exclusive to true on listen

### PR DESCRIPTION
**Pull request by fengmk2**
make sure handle is not shared on cluster module

##### Checklist

- [ ] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

### CODE REVIEW

**Comment by fengmk2** [comment]
https://github.com/node-modules/zookeeper-cluster-client 依赖这个

File: test/cluster.test.js [hunk 1]
```diff
@@ -69,4 +69,15 @@ describe('test/cluster.test.js', () => {
       .expect('stdout', /success/)
       .end(done);
   });
+
+  it('should work on cluster module', () => {
+    return coffee.fork(path.join(__dirname, 'supports/cluster_server.js'))
+      // .debug()
+      // make sure leader and follower exists
+      .expect('stdout', /, leader: true/)
+      .expect('stdout', /, leader: false/)
```
Comment (gxcsoccer) [inline test/cluster.test.js]: 这样测还比较方便

**Comment by gxcsoccer** [comment]
windows 用例不稳定

**Comment by gxcsoccer** [comment]
node 7 不用测试了吧

**Comment by fengmk2** [comment]
@gxcsoccer 去掉 node 7 测试了。

File: test/cluster.test.js (hunk 1 above)
Comment (fengmk2) [inline test/cluster.test.js]: coffee 绝对是进程测试能手 @popomore

Comment (popomore) [inline test/cluster.test.js]: 如果你看到 tf 的实现，你会惊叹

**Comment by fengmk2** [comment]
@gxcsoccer 可以合并发布了。

**Comment by gxcsoccer** [comment]
1.6.6

**Pull request merged**